examples/src/main/java/ninja/soroosh/chatopia/core/example/MessageEchoExample.java

```diff
@@ -62,6 +62,11 @@ public Response onPhCommand(String message, Context context) throws MalformedURL
                 .withCaption("chetoram?");
     }
 
+    @OnCommand(value = "video", help = "A showcase to send a video")
+    public Response onVideoCommand(String message, Context context) {
+        return Response.asVideo(this.getClass().getClassLoader().getResourceAsStream("thankyounote.mp4"));
```

Comment:
maybe a smaller mp4 file? 
This one is ~9.5MB, a bit too big to clone with low speed internet.